src/github/client/github-installation-client.ts

```diff
@@ -94,9 +94,19 @@ export class GitHubInstallationClient extends GitHubClient {
 	 * Get publicly available information for user with given username.
 	 */
 	public getUserByUsername = async (username: string): Promise<AxiosResponse<Octokit.UsersGetByUsernameResponse>> => {
-		return await this.get<Octokit.UsersGetByUsernameResponse>(`/users/{username}`, {}, {
+		const response = await this.get<Octokit.UsersGetByUsernameResponse>(`/users/{username}`, {}, {
 			username
 		});
+		if (response.status === 200 && response.data) {
```

Comment:
why to log this info?

Reply:
ah sorry, that's another thing (just decided to merge it in one go). Gathering impact for Saiyans